### ISSUE TITLE
Syntax Highlighting

### ISSUE THREAD

**Issue by skipkayhil** [skipkayhil/skipkayhil.github.io#12]
The current Syntax Highlighting theme (minima dark) is not great. It currently fails lighthouse accessibility with "Background and foreground colors do not have a sufficient contrast ratio." (The comment highlighting in particular)

The current plan is to replace it with a base16 theme from Rouge:

### `base16-rouge`
- [x] Created https://github.com/skipkayhil/base16-rouge
- [ ] Update `base16-rouge` once the `base16` theme is fixed up
- [x] Last time I built with my fork, but opened tinted-theming/base16-builder-ruby#9

### `rouge`
- [ ] rouge-ruby/rouge#1925
